### ISSUE TITLE
Scss compiler doesn't find align-self or display-flex mixing

### ISSUE THREAD

**Issue by YannDanthu** [francocatena/meteor-compass#10]
Hi,

My scss files are in a package configured as documented.

I got the following error with display-flex or align-self mixin
`Scss compiler error: no mixin named align-self`

But works for mixin background-size.

Any idea ?
Cheers,
Yann

**Comment by francocatena** [comment]
Hi @YannDanthu,

Yes, it is because the version included in this package doesn't have the flexbox support yet. Mostly because of the Sass features used by the Compass folks are not yet implemented by the libsass folks.

I have a branch with the latest Compass release, but it doesn't work properly yet. As soon as it does I'll back to you and let you know.

I'll keep this issue open until thats done.

Cheers.

**Comment by YannDanthu** [comment]
ok thanks for the info.
In the meantime we keep our grunt script to generate the css then ;)

When you say libsass your talking about https://github.com/sass/libsass ?

**Comment by francocatena** [comment]
Exactly, they are slightly behind the Ruby implementation (the one used by the Compass folks), but catching up very quickly.

The most common Meteor wrapper for this lib is https://github.com/fourseven/meteor-scss (the one suggested on the README)

**Comment by YannDanthu** [comment]
That's the one I use as written in the README.md ;)
I will wait for the update as I may not be able to contribute to speedup the integration of compass :laughing:

**Comment by Kabangi** [comment]
Am experiencing the same issue. Waiting for your update on this. 

Thanks for your hard work.
